pages/admin.vue

```diff
@@ -14,10 +14,122 @@
       return navigateTo(`${to.fullPath}/companies`)
     },
   })
+
+  const route = useRoute()
+  const appStore = useAppStore()
+
+  const currentCategory = computed(() => {
+    const routeNames = route.fullPath.split('/')
+    return routeNames[routeNames.length - 1]
+  })
+
+  const categories = [
+    {
+      name: 'companies',
+      icon: 'tag-outline',
```

Comment:
Could possibly change this to `domain` or `office-building`? The tag I had in the Figma design looks a bit wonky in hind sight :)